### ISSUE TITLE
Add light.randomColor 

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt-adafruit#433]
I think it would be useful to add that to our library of colors.

It would be cool to just have a block that returns a random color.

@pelikhan @Jaqster thoughts?

**Comment by microsoftsam** [comment]
Right now I'm thinking something that wraps this: 
```
function randomColor() {
    let rand = Math.randomRange(0, 255);
    return light.hsv(rand, 255, 255);
}
```

**Comment by pelikhan** [comment]
Sounds good to me. Maybe we should limit to a palette of clearly différentiable colors.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by microsoftsam** [comment]
Yeah. If only there was a way to determine that programatically..

**Comment by Jaqster** [comment]
I like it.  Would be even cooler if we had a generic Random block/function built off our Random Math block that could output random light colors and random music tones as well...
